experimental/packages/opentelemetry-instrumentation/test/node/EsmInstrumentation.test.mjs

```diff
@@ -15,12 +15,12 @@
  */
 
 import * as assert from 'assert';
-import {
-  InstrumentationBase,
-  InstrumentationNodeModuleDefinition,
-} from '../../build/src/index.js';
+import * as mod from '../../build/src/index.js';
 import * as exported from 'test-esm-module';
 
+const InstrumentationBase = mod.InstrumentationBase;
+const InstrumentationNodeModuleDefinition = mod.InstrumentationNodeModuleDefinition
```

Comment:
This is a workaround for https://github.com/DataDog/import-in-the-middle/issues/29